policyengine_us/parameters/gov/states/ny/tax/income/credits/household_credit/non_single/base.yaml

```diff
@@ -28,6 +28,10 @@ brackets:
       1986-01-01: 60
   - threshold:
       1986-01-01: 20_000
```

Comment:
```suggestion
      1986-01-01: 22_000
```

Reply:
Would something like this work? (We are basically extending the $60 bracket right?)